cypress/integration/NewLexemeForm.spec.js

```diff
@@ -1,21 +1,78 @@
 describe( 'NewLexemeForm', () => {
 
-	it( 'submits form data', () => {
+	it( 'submits form data with inferred language code', () => {
 		cy.visit( '/' );
 
 		cy.on( 'window:alert', cy.stub().as( 'alert' ) );
 
+		cy.intercept( { query: { action: 'wbgetclaims' } }, {
+			claims: {
+				P218: [ {
+					mainsnak: {
+						snaktype: 'value',
+						property: 'P218',
+						hash: 'faa942355921695b9fa30805f0bb66bc7ed5a433',
+						datavalue: {
+							value: 'en-gb',
+							type: 'string',
+						},
+						datatype: 'external-id',
+					},
+					type: 'statement',
+					id: 'Q9301$6e2ffdaf-4ec3-055f-7165-4849df92ffde',
```

Comment:
This statement ID doesn’t match the language item ID below, which I think could cause confusion later… I suggest we invent a fresh statement ID.
```suggestion
					id: 'Q123$c5c5b451-38a0-4d7c-99b9-af4fdf893d08',
```

Reply:
Fixed thanks.